pkg/util/namespace.go

```diff
@@ -57,14 +57,6 @@ func EnsureNamespaceExist(client kubeclient.Interface, namespace string, dryRun
 		return namespaceObj, nil
 	}
 
-	exist, err := IsNamespaceExist(client, namespace)
```

Comment:
`func IsNamespaceExist()` could be removed.

Reply:
Hi @lonelyCZ and @jwcesign , we ran into issue after this check got removed. It just keeps creating namespaces. Could you fix it or simply add it back? Thank you

Reply:
Hi @yizhang-zen, thanks for spotting this problem. How to reproduce this problem?